packages/app/src/components/Sidebar/PageTree/Item.tsx

```diff
@@ -109,6 +110,15 @@ const Item: FC<ItemProps> = (props: ItemProps) => {
 
   const { data, error } = useSWRxPageChildren(isOpen ? page._id : null);
 
+
+  const [{ isDragging }, drag] = useDrag(() => ({
+    type: 'PAGE_TREE',
+    item: { page },
+    collect: monitor => ({
+      isDragging: !!monitor.isDragging(),
+    }),
+  }));
+
```

Comment:
## Reference
- [useDrag](https://react-dnd.github.io/react-dnd/docs/api/use-drag#usedrag)
- [Items and Types](https://react-dnd.github.io/react-dnd/docs/overview#items-and-types)